{{cookiecutter.repo_name}}/docs/installation.rst

```diff
@@ -6,34 +6,73 @@
 Installation
 ============
 
-
 Stable release
 --------------
 
-To install {{ cookiecutter.project_name }}, run this command in your terminal:
+Using pip
++++++++++
+
+Python comes with an inbuilt package management system, `pip`_. Pip can install, update, or delete any official package.
+
+To install {{ cookiecutter.project_name }} using pip, run this command in your terminal:
 
 .. code-block:: console
 
     $ pip install git+ssh://git@github.com/anmut-consulting/{{ cookiecutter.repo_name }}.git@latest
 
-This is the preferred method to install {{ cookiecutter.project_name }}, as it will always install the most recent stable release.
+This will always install the latest version of `{{ cookiecutter.repo_name }}` provided you have `SSH set up on GitHub`_.
+
+To activate the environment (assuming you are using one), run this command from the root directory of the repo::
+
+    $ source env/bin/activate
+
+And to deactivate the environment::
+
+    $ deactivate
+
+If you don't have pip installed, these `Python installation guides`_ can guide you through the process depending on your operating system and which python version you need.
+
+.. _pip: https://pip.pypa.io/en/stable/
+.. _SSH set up on GitHub: https://help.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh
```

Comment:
Ah, I see you moved the SSH setup to here.  Okay ignore my comment about leaving it in the other file.